cmd/authd/daemon/daemon.go

```diff
@@ -54,6 +54,9 @@ func New() *App {
 		Long:/*i18n.G(*/ "Authentication daemon bridging the system with external brokers.", /*)*/
 		Args:                                                                                cobra.NoArgs,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			// First thing, initialize the journal handler
+			log.InitJournalHandler()
```

Comment:
Ok, better.

Maybe in `internal/services/withexamples.go` can now just do `log.SetHandler(nil)`, so we just continue using normal logging there?

Reply:
How about we add a `--no-journal` option which you can use when you want authd to print to stderr (with or without the example broker)?

Reply:
I'm now using `journal.StderrIsJournalStream` to decide whether to print to the journal or to stderr. Does that handle your use case of running the example broker?

Reply:
It's fine for the app, in the PAM case though (being a library or an executable launched by a library), I think we should just always use the journal if no other option is available (as we were doing before), because otherwise we'd likely log nothing.

So imho the `journal.StderrIsJournalStream()` check has to be moved at daemon level or just having two different init functions.

Reply:
Good point. Done.